### ISSUE TITLE
Update access-limited status of Whitehall attachments in Asset Manager

### ISSUE THREAD

**Issue by chrisroos** [alphagov/asset-manager#500]
We'll need to do this once we're confident that we're setting the access-limited state of all new Whitehall attachments in Asset Manager (https://github.com/alphagov/asset-manager/issues/450).

**Comment by floehopper** [comment]
Blocked by https://github.com/alphagov/asset-manager/issues/507.

**Comment by chrisroos** [comment]
Superseded by #525. Closing.
